app/assets/stylesheets/budget.css

```diff
@@ -0,0 +1,321 @@
+  border-radius: 4px;
+  color: var(--white_whale);
+  display: inline-block;
+  font-size: var(--default_font_size);
+  margin-left: 80px;
+  padding: 2px 7px 2px 7px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.button a:visited {
+  background-color: var(--sacred_beam);
+  border: var(--default_border) var(--sacred_beam);
+  color: var(--white_whale);
+}
+
+.dropdown {
+  float: left;
+  overflow: hidden;
+}
+
+.dropdown .dropbtn {
+  background-color: inherit;
+  border: none;
+  color: var(--white_whale);
+  font-family: inherit;
+  font-size: var(--default_font_size);
+  margin: 0;
+  outline: none;
+  padding: 14px 16px;
+}
+
+.navbar a:hover, .dropdown:hover .dropbtn {
+  background-color: var(--now_what);
+  color: var(--sacred_beam)
+}
+
+.dropdown-content {
+  background-color: var(--sacred_beam);
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  display: none;
+  position: absolute;
+  width: 120px;
+  z-index: 1;
+}
+
+div .dropdown-content a {
+  color: var(--white_whale);
+  display: block;
+  float: left;
+  font-size: 18px;
+  margin: -10px;
+  padding-bottom: 10px;
+  text-align: left;
+  text-decoration: none;
+}
+
+.dropdown-content a:hover {
+  background-color: var(--now_what);
+}
+
+.dropdown:hover .dropdown-content {
+  display: block;
+}
+
+.navbar {
+  background-color: var(--sacred_beam);
+  overflow: hidden;
+}
+
+.navbar a {
+  color: var(--white_whale);
+  float: left;
+  font-size: var(--default_font_size);
+  padding: 14px 16px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.dropdown {
+  float: left;
+  overflow: hidden;
+}
+
+.dropdown .dropbtn {
+  background-color: inherit;
+  border: none;
+  color: var(--white_whale);
+  font-family: inherit;
+  font-size: var(--default_font_size);
+  margin: 0;
+  outline: none;
+  padding: 14px 16px;
+}
+
+.navbar a:hover, .dropdown:hover .dropbtn {
+  background-color: var(--now_what);
+  color: var(--sacred_beam)
+}
+
+.dropdown-content {
+  background-color: var(--sacred_beam);
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  display: none;
+  position: absolute;
+  width: 120px;
+  z-index: 1;
+}
+
+div .dropdown-content a {
+  color: var(--white_whale);
+  display: block;
+  float: left;
+  font-size: 18px;
+  margin: -10px;
+  padding-bottom: 10px;
+  text-align: left;
+  text-decoration: none;
+}
+
+.dropdown-content a:hover {
+  background-color: var(--now_what);
+}
+
+.dropdown:hover .dropdown-content {
+  display: block;
+}
+
+.navbar {
+  background-color: var(--sacred_beam);
+  overflow: hidden;
+}
+
+.navbar a {
+  color: var(--white_whale);
+  float: left;
+  font-size: var(--default_font_size);
+  padding: 14px 16px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.flashes #flash-alert {
+  background-color: var(--desert_heat);
+  border-top: var(--default_border) var(--golden_desert);
+  border-bottom: var(--default_border) var(--golden_desert);
+  color: var(--sacred_beam);
+  display: block;
+  text-align: center;
```

Comment:
There is a lot of repetition here. Is there a way to remove the repeated rules?

Reply:
Yep, thanks for the pickup, removed display and text-align from the four flashes to one .flashes styling, ad-css-styling sha: 48e905b